.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@cowboyd if you like, i can remove this file

Reply:
I think it's actually great